middle_end/flambda2/types/expand_head.mli

```diff
@@ -102,6 +102,12 @@ val is_bottom : Typing_env.t -> Type_grammar.t -> bool
 
 val is_unknown : Typing_env.t -> Type_grammar.t -> bool
 
+val reduce_is_int :
+  Typing_env.t -> Type_grammar.t -> Targetint_31_63.Set.t Or_unknown_or_bottom.t
+
+val reduce_get_tag :
+  Typing_env.t -> Type_grammar.t -> Targetint_31_63.Set.t Or_unknown_or_bottom.t
+
```

Comment:
```suggestion
val reduce_is_int :
  Typing_env.t -> block_type:Type_grammar.t -> Targetint_31_63.Set.t Or_unknown_or_bottom.t

val reduce_get_tag :
  Typing_env.t -> block_type:Type_grammar.t -> Targetint_31_63.Set.t Or_unknown_or_bottom.t

```
I would add a label to those arguments to make it obvious what is expected. Otherwise we could imagine that the type agument might be the integer returned by `is_int` or `get_type`.

Reply:
By the way those functions are not used. They should be called from the meet.